test/jest/victory-errorbars/victory-errorbars.test.js

```diff
@@ -0,0 +1,1087 @@
+          });
+        });
+
+        it("should check bottom border of error bars positions", () => {
+          const svgDimensions = { width: 350, height: 200, padding: 75 };
+          const borderWidth = 10;
+          const data = [
+            { x: 0, y: 0, errorX: 0.1, errorY: 0.2 },
+            { x: 2, y: 3, errorX: 0.1, errorY: 0.2 },
+            { x: 5, y: 5, errorX: 0.1, errorY: 0.2 }
+          ];
+          render(
+            <VictoryErrorBar
+              data={createData(data)}
+              borderWidth={borderWidth}
+              {...svgDimensions}
+              {...defaultProps}
+            />
+          );
+
+          const xScale = d3Scale
+            .scaleLinear()
+            .domain([-0.1, 5.1])
+            .range([
+              svgDimensions.padding,
+              svgDimensions.width - svgDimensions.padding
+            ]);
+
+          const yScale = d3Scale
+            .scaleLinear()
+            .domain([-0.2, 5.2])
+            .range([
+              svgDimensions.height - svgDimensions.padding,
+              svgDimensions.padding
+            ]);
+
+          const bars = screen.getAllByTestId("error-bar");
+          bars.forEach((node, i) => {
+            const errorY = yScale(data[i].y + data[i].errorY);
+            const yScaleMin = yScale.range()[1];
+            const negativeErrorY = errorY <= yScaleMin ? yScaleMin : errorY;
+
+            const [x1, x2, y1, y2] = getCoordinatesForLineWithType(
+              node,
+              "border-bottom"
+            );
+
+            expect(x1).toEqual(xScale(data[i].x) - borderWidth);
+            expect(x2).toEqual(xScale(data[i].x) + borderWidth);
+            expect(y1).toEqual(negativeErrorY);
+            expect(y2).toEqual(negativeErrorY);
+          });
+        });
+
+        it("should check top border of error bars positions", () => {
+          const svgDimensions = { width: 350, height: 200, padding: 75 };
+          const borderWidth = 10;
+          const data = [
+            { x: 0, y: 0, errorX: 0.1, errorY: 0.2 },
+            { x: 2, y: 3, errorX: 0.1, errorY: 0.2 },
+            { x: 5, y: 5, errorX: 0.1, errorY: 0.2 }
+          ];
+          render(
+            <VictoryErrorBar
+              data={createData(data)}
+              borderWidth={borderWidth}
+              {...svgDimensions}
+              {...defaultProps}
+            />
+          );
+
+          const xScale = d3Scale
+            .scaleLinear()
+            .domain([-0.1, 5.1])
+            .range([
+              svgDimensions.padding,
+              svgDimensions.width - svgDimensions.padding
+            ]);
+
+          const yScale = d3Scale
+            .scaleLinear()
+            .domain([-0.2, 5.2])
+            .range([
+              svgDimensions.height - svgDimensions.padding,
+              svgDimensions.padding
+            ]);
+
+          const bars = screen.getAllByTestId("error-bar");
+          bars.forEach((node, i) => {
```

Comment:
We should probably always use a `.toHaveLength(...)` assertion, so that we're sure the following assertions are running.  Or maybe `.not.toHaveLength(0)` if the quantity doesn't matter.

Reply:
Agreed, or an `expect.hasAssertions()` to verify that the `forEach` has at least one element.